WeakAuras/Types.lua

```diff
@@ -1320,6 +1320,66 @@ for id, str in pairs(Private.combatlog_spell_school_types) do
   Private.combatlog_spell_school_types_for_ui[id] = ("%.3d - %s"):format(id, str)
 end
 
+Private.GetCurrencyListInfo = function(index)
+  if WeakAuras.IsRetail() then
+    return C_CurrencyInfo.GetCurrencyListInfo(index)
+  elseif WeakAuras.IsWrathClassic() then
+    local name, isHeader, isExpanded, isUnused, isWatched, _, icon, _, hasWeeklyLimit, _, _, itemID = GetCurrencyListInfo(index)
+    if itemID then
+      local _, currentAmount, _, earnedThisWeek, weeklyMax, totalMax, isDiscovered, rarity = GetCurrencyInfo(itemID)
+      local currencyInfo = {
+        name = name,
+        description = "",
+        isHeader = isHeader,
+        isHeaderExpanded = isExpanded,
+        isTypeUnused = isUnused,
+        isShowInBackpack = isWatched,
+        quantity = currentAmount,
+        trackedQuantity = 0,
+        iconFileID = icon,
+        maxQuantity = totalMax,
+        canEarnPerWeek = hasWeeklyLimit,
+        quantityEarnedThisWeek = earnedThisWeek,
+        isTradeable = false,
+        quality = rarity,
+        maxWeeklyQuantity = weeklyMax,
+        totalEarned = 0,
+        discovered = isDiscovered,
+        useTotalEarnedForMaxQty = false,
+      }
+      return currencyInfo
+    end
+  end
+end
+
+Private.GetCurrencyListSize = function()
+  if WeakAuras.IsRetail() then
+    return C_CurrencyInfo.GetCurrencyListSize()
+  end
+  if WeakAuras.IsWrathClassic() then
+    return GetCurrencyListSize()
+  end
+end
+
+Private.ExpandCurrencyList = function(index, expand)
+  if WeakAuras.IsRetail() then
+    return C_CurrencyInfo.GetCurrencyListLink(index, expand)
```

Comment:
this seems wrong, shouldn't it be C_CurrencyInfo.ExpandCurrencyList?

Reply:
yes. 👀